helix-term/src/commands.rs

```diff
@@ -5826,9 +5838,12 @@ fn shell_prompt(cx: &mut Context, prompt: Cow<'static, str>, behavior: ShellBeha
     );
 }
 
-fn suspend(_cx: &mut Context) {
+fn suspend(cx: &mut Context) {
```

Comment:
As @poliorcetics said, let's keep this as `_cx` and use `_cx.block_try_flush_writes()` below to avoid a compiler warning when compiling for Windows